tests/test_ind_indygo_gov_audit.py

```diff
@@ -2,7 +2,7 @@
 from os.path import dirname, join
 
 import pytest
-from city_scrapers_core.constants import BOARD
+from city_scrapers_core.constants import BOARD  # noqa
```

Comment:
See this [comment](https://github.com/City-Bureau/city-scrapers-indianapolis/pull/14/files#r1463728936), re: suppressing linter issues.